basil/HL/sensirion_sht85.py

```diff
@@ -6,17 +6,19 @@
 
 import numpy as np
 import logging
+import struct
 
 from basil.HL.SensirionBridgeDevice import SensirionBridgeI2CDevice
 
 logger = logging.getLogger(__name__)
+logging.getLogger("sensirion_shdlc_driver.connection").setLevel(logging.ERROR)
 
 
 class sensirionSHT85(SensirionBridgeI2CDevice):
     '''
     Driver for the Sensirion SHT85 temperature and humidity sensor.
     Measurements can be performed in three repeatability modes:
-    low (0.15°C 0.21%RH) (default), medium (0.08°C 0.15%RH), high (0.04°C 0.08%RH)
+    low (0.15C 0.21%RH) (default), medium (0.08C 0.15%RH), high (0.04C 0.08%RH)
```

Comment:
is there an actual problem with the `°` sign? if have so far not managed to find any circumstance that produced problems with this?

Reply:
I thing python was complaining on a Windows machine some time ago. At least someone told me.  I did not check again myself, but I can do that.

Reply:
afaik, `cmd` will not display the symbol but it should not crash. making it uglier for everyone just so windows `cmd` doesn't complain isn't fair to my mind ;)

Reply:
Okay... and even if a Windows user experiences some error, he should be capable of removing the ° signs by himself

Reply:
if it actually crashes because of this, it's worth to remove it i guess. would be nice if you could actually check ;)

Reply:
It does not complain on my Windows notebook. Since I do not remember the exact Error someone showed me before, I would  close this, unless it comes up again.